addons/l10n_in/tests/test_l10n_in_fiscal_position.py

```diff
@@ -105,3 +106,33 @@ def test_l10n_in_fiscal_for_branch(self):
             partner=self.partner_foreign,
             taxes=self.igst_sale_18,
         )
+
+    def test_l10n_in_fiscal_in_bill_to_ship_to(self):
```

Comment:
Does this test has a lot of added value?  And the actual case where you set the place of supply manually is not even tested.